integration-tests/src/test/kotlin/org/stellar/anchor/platform/e2etest/Sep24End2EndTests.kt

```diff
@@ -219,11 +214,9 @@ class Sep24End2EndTests : AbstractIntegrationTests(TestConfig(testProfileName =
         .build()
     transfer.sign(keypair)
 
-    submissionLock.withLock {
-      wallet.stellar().submitTransaction(transfer)
-      // Wait for the status to change to PENDING_USER_TRANSFER_END
-      waitForTxnStatus(withdrawTxn.id, COMPLETED, token)
-    }
+    transactionWithRetry { wallet.stellar().submitTransaction(transfer) }
```

Comment:
I think we should keep the lock in addition to the lock
```suggestion
   submissionLock.withLock {
      transactionWithRetry { wallet.stellar().submitTransaction(transfer) }
   }
```